src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -199,24 +198,6 @@ public synchronized static NewGameChooserModel getNewGameChooserModel() {
       return new NewGameChooserModel();
   }
 
-
-  /**
-   * Refreshes the game list (from disk) then caches the new list
-   */
-  private void refreshGameList() {
```

Comment:
Is any of the rest of the functionality in this method needed? Selecting a game?